src/components/ListItems/ListItems.js

```diff
@@ -17,7 +17,7 @@ const ListItems = ({
         text={text}
       />
     ))}
-    <h3>Added items</h3>
+    <h4 className={styles.completedHeading}>Completed</h4>
```

Comment:
I'm a little worried about this text colour standing out against the white:
https://coolors.co/contrast-checker/fabe0e-ffffff

Reply:
Ooh that's an interesting link, will come in handy.

Yes I think you're right, I tried a few different variations and haven't found a good fit yet. This felt like a "for now" decision. I will explore some more ideas and see what I can come up with!